docs/api-reference/aggregation-layers/aggregator.md

```diff
@@ -0,0 +1,168 @@
+# Aggregator Interface
+
+The `Aggregator` interface describes a type of class that performs aggregation.
+
+## Terminology
+
+_Aggregation_ is a 2-step process:
+
+1. **Sort**: Group a collection of _data points_ by some property into bins_.
```

Comment:
Thought for future,: maybe we could define two composable interfaces for these two steps?

Reply:
WebGLAggregator does the two steps in one pass.

Reply:
> WebGLAggregator does the two steps in one pass.

I see. Perhaps one could generalize the code so it could just one of them somehow. For another time...